djgentelella/static/gentelella/js/bootstrap-colorpicker.min.js

```diff
@@ -0,0 +1,5 @@
+/*!
```

Comment:
This file  is not allowed here, also you can not include a externar lib out of vendor folder.
You need to register on devloadstatic command.